### ISSUE TITLE
Cannot read property 'length' of undefinedvalidateQuery

### ISSUE THREAD

**Issue by houshuang** [Lapple/react-json-inspector#28]
This happens when the data prop goes from state: [], to state[{an element}].

``` js
Uncaught TypeError: Cannot read property 'length' of undefinedvalidateQuery @ json-inspector.js?2740:38
```

Version 6.1.2

**Comment by houshuang** [comment]
Changing the invocation to

``` js
const StateView = (state) => <Inspector data={state.state} validateQuery = {(e) => e && e.length > 2}/>
```

completely fixed the problem. Should be easy to fix in source too. Once it's working, it's a great tool!

**Comment by Lapple** [comment]
Thanks, fix landed in 6.1.3.
